tailwind.config.js

```diff
@@ -3,6 +3,9 @@ const plugin = require('tailwindcss/plugin')
 module.exports = {
   theme: {
     extend: {},
+    colors: {
```

Comment:
this is the culprit. It should be inisde `extend`.

Reply:
Weird, just moved it in there and it doesn’t seem to resolve the issue. Did that work for you?

Reply:
Yes. Solved it for me.